run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+while true; do
+	CUDA_VISIBLE_DEVICES=3 python glados.py
+done
```

Comment:
This will definitely need some refinement - not everyone has CUDA devices available, or want to run a massive LLM.

My suggestion would be to take any such params from the config, and maybe even have some automation (e.g. detect if there's a CUDA device that can be used, auto-assign it, etc.).

Reply:
There seems to be a `TTS_USE_CUDA` env var in the config. It could be read here and the handled accordingly

Reply:
Didn't intend to commit this, definitely rushed this draft PR just to let people see what I'm thinking of PR'ing.  Cleanups to come :)